<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RECallRate</class>
 <widget class="QDialog" name="RECallRate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call rate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="EQLineEdit" name="leID">
     <property name="Field" stdset="0">
      <string>f_id</string>
     </property>
     <property name="PrimaryKey" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Direction</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="EQLineEdit" name="leDirection">
     <property name="maxLength">
      <number>1</number>
     </property>
     <property name="Field" stdset="0">
      <string>f_direction</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Local</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="EQLineEdit" name="leLocal">
     <property name="inputMask">
      <string>#</string>
     </property>
     <property name="maxLength">
      <number>1</number>
     </property>
     <property name="Field" stdset="0">
      <string>f_local</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Area</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="EQLineEdit" name="leArea">
     <property name="Field" stdset="0">
      <string>f_area</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Area code</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="EQLineEdit" name="leCode">
     <property name="Field" stdset="0">
      <string>f_code</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Rate</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="EQLineEdit" name="leRate">
     <property name="Field" stdset="0">
      <string>f_rate</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>114</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>113</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
